### ISSUE TITLE
Release notes?

### ISSUE THREAD

**Issue by onnozweers** [gridcf/gct#164]
Dear people,

Are there any release notes? Currently it's very hard for me (as a non-developer) to see whether patch #155 has been released in this version:

```
[surfadvisors-ozweers@ui-02 ~]$ globus-url-copy -version
globus-url-copy: 10.9
```

Cheers,
Onno

**Comment by fscheiner** [comment]
@msalle 
> The release notes for the next gct does list the individual versions in the list of tarballs. We might consider adding those versions also at the text describing the changes @fscheiner what do you think?

Good idea. I added the version numbers on the release page but missed to get back to you about that.

@onnozweers 
Does [the release page of our current release](https://github.com/gridcf/gct/releases/tag/v6.2.20210826) give all the info you need?

For example the fix for #155 is in **globus-gssapi-gsi** version `14.17`, so even effective with older versions of `globus-url-copy` as long as that specific version of **globus-gssapi-gsi** is in use.

**Comment by onnozweers** [comment]
>  Does the release page of our current release give all the info you need?

Yes, awesome! Thanks!
